locales/en/messages.json

```diff
@@ -3242,10 +3242,10 @@
         "message": "Firmware not loaded"
     },
     "firmwareFlasherFirmwareLocalLoaded": {
-        "message": "Loaded Local Firmware: ($1 bytes)"
+        "message": "Loaded Local Firmware: $1 ($2 bytes)"
```

Comment:
Now that you are changing it, better to use the ”object" way with {{firmwareSelected}} and {{size}} for example. I think is better for understanding the code and helps translators.